### ISSUE TITLE
CLA Assistant asking for a CLA from a member of an exempt organization

### ISSUE THREAD

**Issue by egrefen** [cla-assistant/cla-assistant#954]
In the following pull request: https://github.com/cohere-ai/cohere-python/pull/133, user @sanderland is being asked to sign a CLA despite being part of the [cohere-ai](https://github.com/cohere-ai) organization
<img width="405" alt="Screenshot 2023-01-11 at 09 14 02" src="https://user-images.githubusercontent.com/1145770/211766261-b91a31f1-c741-4d4c-8755-be6ce7125943.png">
which is whitelisted in our CLA setup
<img width="571" alt="Screenshot 2023-01-11 at 09 13 32" src="https://user-images.githubusercontent.com/1145770/211766329-aaf2a264-6789-4557-b7ae-5c1fcb25c063.png">
Other members of our org have been able to submit PRs without being prompted for a CLA so not sure what is happening here.

**Comment by KharitonOff** [comment]
@egrefen CLA assistant can only recognize the public members of the org. It looks like @sanderland hasn't made his membership public https://github.com/orgs/cohere-ai/people

**Comment by RafalSkolasinski** [comment]
@KharitonOff I have similar situation in here https://github.com/SeldonIO/seldon-core/pull/5509
I am member of SeldonIO organization (public member as far as I am aware) and I am being asked to sign the CLA 

![image](https://github.com/cla-assistant/cla-assistant/assets/10928117/8c7f8bac-76ba-4a91-9a50-e8109fa12fe8)

**Comment by KharitonOff** [comment]
@RafalSkolasinski you are not a public member of the SeldonIO (otherwise the organization would be listed in your public profile). You need to make your membership public as described here https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership

**Comment by RafalSkolasinski** [comment]
You are correct @KharitonOff - there was confusion on my side - all works as expected. 

QQ as we are on the subject -> are there plans for it to also work with private memberships?

**Comment by KharitonOff** [comment]
@RafalSkolasinski No, as far as I know, there are no such plans.
